### PR TITLE
Update params.json

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: params.json [hunk 1]
```diff
@@ -1 +1 @@
...or and switch to another theme. Your Page content will be preserved if it remained markdown format.\r\n\r\n### Rather Drive Stick?\r\nIf you prefer to not use the automatic generator, push a branch named `gh-pages` to your repository to create a page manually. In addition to supporting regular HTML content, GitHub Pages support Jekyll, a simple, blog aware static site generator written by our own Tom Preston-Werner. Jekyll makes it easy to create site-wide headers and footers without having to copy them across every page. It also offers intelligent blog support and other advanced templating features.\r\n\r\n### Authors and Contributors\r\nYou can @mention a GitHub username to generate a link to their profile. The resulting `<a>` element will link to the contributor's GitHub Profile. For example: In 2007, Chris Wanstrath (@defunkt), PJ Hyett (@pjhyett), and Tom Preston-Werner (@mojombo) founded GitHub.\r\n\r\n### Support or Contact\r\nHaving trouble with Pages? Check out the documentation at http://help.github.com/pages or contact support@github.com and we’ll help you sort it out.\r\n","google":"","note":"Don't delete this file! It's used internally to help with page regeneration."}
```
Comment (Aprianto96) [inline params.json:1]: aprianto96

Comment (Tyemg) [reply params.json:1]: Create .github/workflows/main.yml

Comment (Tyemg) [reply params.json:1]: github/workflows/main.yml

**Comment by Reggaevibes** [comment]
Looking good

lør. 15. apr. 2023, 08:49 skrev Tyemg ***@***.***>:

**Comment by rukiye95n** [comment]
Waaaaaaaaayبتاريخ ١٥‏/٠٤‏/٢٠٢٣ ١٠:٠٦ ص، كتب Reggaevibes ***@***.***>:
Looking good

lør. 15. apr. 2023, 08:49 skrev Tyemg ***@***.***>:

**Comment by perrystan1489** [comment]
Hello world

File: params.json (hunk 1 above)
Comment (perrystan1489) [reply params.json:1]: Can someone check my workflow

**Pull request closed without merging**